index.js

```diff
@@ -0,0 +1,38 @@
+(function() {
+
+  const Actuate = animations => $ => new Promise ((resolve, reject) => {
+
+    const commands = Array.isArray(animations) ?
+      animations : animations.split(' ')
+
+    const event = [
+      'animationsend',
+      'webkitAnimationEnd',
+      'oAnimationEnd',
```

Comment:
nice.. perhaps this would achieve the same thing then?
```
    const types = {
      'transition':'transitionend',
      'OTransition':'oTransitionEnd',
      'MozTransition':'transitionend',
      'WebkitTransition':'webkitTransitionEnd'
    }
   const event = types.find(x => document.body.style[t] !== undefined)
```